GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategory.gi

```diff
@@ -42,8 +42,13 @@ InstallMethod( AssociatedMorphism,
                [ IsGeneralizedMorphism ],
                
   function( generalized_morphism )
+    local associated_morphism;
     
-    return DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    associated_morphism := DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    
+    INSTALL_TODO_LIST_FOR_GENERALIZED_MORPHISMS( associated_morphism, generalized_morphism );
```

Comment:
Actually, the propagation of
`IsMonomorphism`, `IsEpimorphism`, and `IsIsomorphism`
in one direction (namely from the generalized morphism to its associated morphism)
is correct :)
So `INSTALL_TODO_LIST_FOR_GENERALIZED_MORPHISMS`
should install this particular direction.